tlib/src/system.cpp

```diff
@@ -108,7 +109,8 @@ void tlib::reboot(){
     __builtin_unreachable();
 }
 
-void tlib::shutdown(){
+void tlib::shutdown(int delay=0){
+    tlib::sleep_ms(1000* delay);
```

Comment:
Could you add a test to do the sleep only if delay is not zero? And add a space before *.